RBDM/Double.h

```diff
@@ -0,0 +1,20 @@
+#ifndef Double_h_
+#define Double_h_
+
+#include"Base.h"
+
+class Double : public BaseForType
+{
+private:
+	double value;
+public:
+	Double();
+	Double(double &value);
+	Double(const Double &value);
+	void Set(const double value);
```

Comment:
Two questions:
1) Why do we need two Set functions here?
2) Why do we need Set() function in base class if we don't override it here?